### PR TITLE
Fix Bugs (Patch_9)

### PR DESCRIPTION
**Pull request by NekoIceCream**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -41,7 +40,7 @@ client.categories = fs.readdirSync("./commands/");
 ["command"].forEach(handler => {
     require(`./handlers/${handler}`)(client);
 }); 
-ready.on('ready', () => {
+client.on('ready', () => {
```
Comment (GameWatch21) [inline index.js]: `client.once`

**Review by GameWatch21 — COMMENTED** [review]
Nih

File: index.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ client.player = player;
         const event = require(`./events/${file}`);
         let eventName = file.split(".")[0];
         console.log(`Loading event ${eventName}`);
-        client.on(eventName, event.bind(null, client));
+        client.once(eventName, event.bind(null, client));
```
Comment (GameWatch21) [inline index.js:20]: `client.player.once`

**Review by GameWatch21 — COMMENTED** [review]
Nih

File: events/chatbot.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const { chatBot } = require("reconlx");
 const client = require("../index");
 const schema = require("../models/chatbot-channel");
-client.on("message", async (message) => {
+client.once("message", async (message) => {
```
Comment (GameWatch21) [inline events/chatbot.js:4]: gk boleh ada dua `client.once`

Comment (GameWatch21) [reply events/chatbot.js:4]: kalau mau, gabungin ke index.js

**Review by GameWatch21 — COMMENTED** [review]
Nih

**Comment by NekoIceCream** [comment]
Fixed by @GameWatch21

**Pull request closed without merging**